graylog2-web-interface/src/logic/permissions/GRN.js

```diff
@@ -25,7 +25,7 @@ export const getShowRouteFromGRN = (grn: string) => {
       return Routes.dashboard_show(id);
     case 'event_definition':
       return Routes.ALERTS.DEFINITIONS.edit(id);
-    case 'event_notification':
+    case 'notification':
```

Comment:
Is this change intended?

Reply:
oops, no. but after a merge it will be gone. It's in master already

Reply:
Okay :+1: